runtime/Jakt/String.h

```diff
@@ -19,6 +19,11 @@
 
 namespace Jakt {
 
+bool operator<(char const* characters, String const& string);
```

Comment:
What code complains when these aren't here? Where are they implemented?

Reply:
This error for each of them:

runtime/Jakt/String.cpp:174:6: error: no previous declaration for 'bool Jakt::operator<(const char*, const String&)' [-Werror=missing-declarations]